loader/full-struct_test.go

```diff
@@ -217,10 +217,24 @@ func services(workingDir, homeDir string) types.Services {
 			Ipc:      "host",
 			Uts:      "host",
 			Labels: map[string]string{
+				"FOO":                     "foo_from_label_file",
```

Comment:
FYI this huge test we inherited from legacy compose parser in docker/cli.
As it is painful to maintain, we prefer to have individual test checking specific attribute is well supported (which you also provided 🥳)

Reply:
should I revert these changes or keep them?